.gitattributes

```diff
@@ -1,6 +1,14 @@
-/tests export-ignore
-/spec export-ignore
-/.travis.yml export-ignore
-/.gitignore export-ignore
-/.gitattributes export-ignore
-/phpunit.xml.dist export-ignore
```

Comment:
Why not exclude phpunit config file?

Reply:
I probably copied the file from another plugin where no phpunit config file was present. I have added it now :)